windows_containers/windows-containers-release-notes-5-x.adoc

```diff
@@ -21,13 +21,21 @@ ifndef::openshift-origin[]
 
 {productwinc} is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription. It requires an additional Red Hat subscription and is supported according to the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
 
-You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+You must have this separate subscription to receive support for Windows Container Support for Red Hat OpenShift. Without this additional Red Hat subscription, deploying Windows container workloads in production clusters is not supported. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
 For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[Red Hat OpenShift support for Windows Containers].
 
-If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support. 
+If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
```

Comment:
You could potentially split this line into two to facilitate localisation and to then turn the second element into a note. I also replaced the work "lack" to explicitly state that the community version is not supported by Red Hat. For example:

> If you do not have this additional Red Hat subscription, you can alternatively use the Community Windows Machine Config Operator.
>
> **Note**: Red Hat does not provide support for the Community Windows Machine Config Operator.

Reply:
You could also potentially provide a link to further reading on the Community Windows Machine Config Operator.